tensorflow/training/docker/2.9/py3/Dockerfile.cpu

```diff
@@ -263,14 +263,15 @@ RUN HOROVOD_WITH_MPI=1 \
    && ${PIP} install --no-cache-dir horovod==0.24.2
 
 RUN $PYTHON -m pip install --no-cache-dir -U \
+    numba \
```

Comment:
Curious to know why we had a version constraint for `numba` in the first place?

Reply:
Might have been a constraint on the numpy version available!